client/tests/integration/manuscript-manager-template-test.coffee

```diff
@@ -23,7 +23,10 @@ module 'Integration: Manuscript Manager Templates',
   setup: ->
     app = startApp()
     server = setupMockServer()
-
+    Tahi.__container__.lookup(
+      'controller:admin/journal/manuscript-manager-template/edit'
+    )._actions.saveTemplateOnClick = -> console.log 'No Action'
+      
```

Comment:
I couldn't find a better way to handle this.

- I tried sinon again to no avail.
- I tried to preemptively navigate away from the page in order to avoid the request being triggered in `teardown`.  The tests, though, are so quick, so the records were still in flight when `teardown` is eventually called.